arlas-tagger-core/src/main/java/io/arlas/tagger/app/KafkaConfiguration.java

```diff
@@ -45,4 +48,16 @@
 
     @JsonProperty("kafka-topic-execute-tags")
     public String executeTagsTopic;
+
+    @JsonProperty("kafka-extra-properties")
+    private String kafkaExtraProperties;
+
+    private Map<String, String> extraProperties;
```

Comment:
simplify by removing extraProperties and just return `StringToMap.parse(kafkaExtraProperties); ` in `getExtraProperties `